docs/tutorial/tasks.md

```diff
@@ -25,8 +25,8 @@ Flows and tasks share some common features:
 
 Network calls (such as our `GET` requests to the GitHub API) are particularly useful as tasks because they take advantage of task features such as [retries](/concepts/tasks/#retries), [caching](/concepts/tasks/#caching), and [concurrency](/concepts/task-runners/#using-a-task-runner).
 
-!!! warning "Tasks must be called from flows"
-    All tasks must be called from within a flow. Tasks may not call other tasks directly.
+!!! warning "Tasks may be called from other tasks"
```

Comment:
Is this still a `warning` admonition? Or is there a happier/less ominous choice (such as `tip` or `info`)

Reply:
good call, make it `tip` in [869bb7a](https://github.com/PrefectHQ/prefect/pull/13154/commits/869bb7aec7a7efc8768e26966d2e6157b32b3b02)